dockerfile/cuda11.1.1.dockerfile

```diff
@@ -24,15 +24,8 @@ RUN apt-get update && \
     libpci-dev \
     libaio-dev \
     libcap2 \
-    libtinfo5
-
-# Install CMake
-RUN wget -q https://github.com/Kitware/CMake/releases/download/v3.17.1/cmake-3.17.1-Linux-x86_64.sh \
-    -O /tmp/cmake-install.sh && \
-    chmod +x /tmp/cmake-install.sh && \
-    mkdir /usr/local/cmake && \
-    /tmp/cmake-install.sh --skip-license --prefix=/usr/local/cmake && \
-    rm /tmp/cmake-install.sh
```

Comment:
Why remove CMake? Would you please write the reason in description?

Reply:
> remove duplicate cmake and ompi

it's already in description

Reply:
I mean the reason.

Reply:
it's already in base image and duplicate here, updated the description